mllib/pom.xml

```diff
@@ -59,6 +59,7 @@
       <groupId>org.jblas</groupId>
       <artifactId>jblas</artifactId>
       <version>${jblas.version}</version>
+     <scope>test</scope>
```

Comment:
Yes I can see skipping the test code for now as its not distributed in an assembly. Eventually that should go too of course. (Nit: indent one more space)